src/main/java/com/palehorsestudios/alone/Main.java

```diff
@@ -0,0 +1,20 @@
+package com.palehorsestudios.alone;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Main {
+  public static void getNarrative(File file) throws FileNotFoundException {
+    Scanner sc = new Scanner(new File(String.valueOf(file)));
+    while(sc.hasNextLine()) {
+      String line = sc.nextLine();
+      System.out.println(line);
+    }
+  }
+  public static void main(String[] args) throws IOException {
```

Comment:
I recommend using a try / catch instead of throwing an IOException in main. That would allow us to print something like: "It seems we have misplaced our narrative. Please come back another time."

Reply:
Will do.